tests/functional/disaster-recovery/metro-dr-pc/test_failover_and_relocate_app.py

```
@@ -0,0 +1,344 @@
+    Test Failover and Relocate actions for MDR using the same workloads
+    (Subscription and ApplicationSet, with RBD or CephFS PVC interface)
+    as used in the RDR test suite.
+    """
+
+    @pytest.fixture(autouse=True)
+    def teardown(self, request, dr_workload):
+        """
+        If fenced, unfence the cluster and reboot nodes
+        """
+
+        def finalizer():
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `finalizer`

Add return type annotation: `None`

(ANN202)